app/controllers/provider_interface/product_page_controller.rb

```diff
@@ -0,0 +1,13 @@
+module ProviderInterface
+  class ProductPageController < ProviderInterfaceController
+    skip_before_action :authenticate_provider_user!, :check_data_sharing_agreements
+
+    layout 'layout'
+
+    def show
+      courses = Course.includes(:provider).visible_to_candidates
```

Comment:
```suggestion
      courses = Course.includes(:provider).applyable
```

`applyable` courses are the courses that are enabled on Apply, `visible_to_candidates` are courses that are visible on Find.

Reply:
Done!

Reply:
https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1166 renames the scopes to be more descriptive, hopefully that makes it clearer.